Quest 7: Can we write try with finally block but not catch block

Ans:-

Yes, It is possible to have a try block without a catch block by using a final block. As we know,
 a final block will always execute even there is an exception occurred in a try block, except System.
  exit() it will execute always.